tests/saw/spec/handshake/s2n_handshake_io.cry

```diff
@@ -170,7 +231,7 @@ IS_TLS13_HANDSHAKE conn = conn.actual_protocol_version == S2N_TLS13
 ACTIVE_STATE_MACHINE : connection -> [20]handshake_action
 ACTIVE_STATE_MACHINE conn = if IS_TLS13_HANDSHAKE conn then tls13_state_machine else state_machine # zero
 
-ACTIVE_HANDSHAKES : connection -> [256][32][32]
+ACTIVE_HANDSHAKES : connection -> [S2N_HANDSHAKES_COUNT][32][32]
```

Comment:
For clarity, why not also include `S2N_MAX_HANDSHAKE_LENGTH`? Take a look at its [definition](https://github.com/aws/s2n-tls/blob/1ff0aa8988e98feb6d106e7551c44f1718974d4a/tls/s2n_handshake_io.c#L140).

Reply:
Ah, good point. I made `S2N_HANDSHAKES_COUNT` a top-level definition mainly so that I could reference it in `initial_connection`, but we could just as well make `S2N_MAX_HANDSHAKE_LENGTH` a top-level definition in addition.